src/morpho-chainlink/MorphoChainlinkOracleV2.sol

```diff
@@ -66,6 +66,8 @@ contract MorphoChainlinkOracleV2 is IMorphoChainlinkOracleV2 {
     /// @param quoteVaultConversionSample The sample amount of quote vault shares used to convert to underlying.
     /// Pass 1 if the quote asset is not a vault. Should be chosen such that converting `quoteVaultConversionSample` to
     /// assets has enough precision.
+    /// @dev Morpho Blue markets have issue when the price of the oracle can go up instantly by a multiplier of more
+    /// than 1/(LLTV*LIF). So in particular, vaults that can receive donations as loan/quote assets are problematic.
```

Comment:
```suggestion
    /// @dev Morpho Blue markets have issue when the price of the oracle can go down instantly by a multiplier of less
    /// than LLTV*LIF. So in particular, vaults that can receive donations as loan/quote assets are problematic.
```
This is the opposite because the oracle in Morpho Blue is pricing the collateral quoted in the loan token, but the problematic case is when the loan token value goes up

Reply:
you are right!